.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: 'Feature Request/Idea:'
+labels: ''
+assignees: ''
+
+---
+<!--
+Thank you for contributing to the Dataverse Project!
```

Comment:
Thank you for contributing to the Dataverse Project through the creation of a feature request!